src/api/rest/server/mcis/manageInfo.go

```diff
@@ -42,8 +42,9 @@ type JSONResult struct {
 // @Param nsId path string true "Namespace ID" default(ns01)
 // @Param mcisId path string true "MCIS ID" default(mcis01)
 // @Param option query string false "Option" Enums(default, id, status, accessinfo)
-// @Param filterKey query string false "(for option==id) Field key for filtering (ex: connectionName)" default(connectionName)
-// @Param filterVal query string false "(for option==id) Field value for filtering (ex: aws-ap-northeast-2)" default(aws-ap-northeast-2)
+// @Param filterKey query string false "(For option==id) Field key for filtering (ex: connectionName)"
+// @Param filterVal query string false "(For option==id) Field value for filtering (ex: aws-ap-northeast-2)"
+// @Param accessInfoOption query string false "(For option==accessinfo) accessInfoOption (showSshKey)"
```

Comment:
REST API 호출 시
http://.../ns/{nsId}/mcis/{mcisId}?**option=id** 와 같이 사용하고 있어서
아래와 같이 제안합니다.. ㅎㅎ
(`option==` 으로 찾아 보면 다른 곳에 3곳 더 있습니다)

```suggestion
// @Param filterKey query string false "(For option=id) Field key for filtering (ex: connectionName)"
// @Param filterVal query string false "(For option=id) Field value for filtering (ex: aws-ap-northeast-2)"
// @Param accessInfoOption query string false "(For option=accessinfo) accessInfoOption (showSshKey)"
```

Reply:
@jihoon-seo 바로 적용하였습니다!